src/Containers/AdminApproval/AdminApproval.js

```diff
@@ -36,7 +44,7 @@ const Admin = () => {
   const allUnvettedDoctors = unvettedDoctorsList?.map(unvetted => {
     return(
       <Card
-        id={`doctor-${unvetted.id}`}
```

Comment:
The two separate tables for doctors and mhps have incremental ids when new ones are created, which means that across tables, sometimes they have the same id number. The interpolation was there to make sure each card had a unique id, so I think we want to revert that bit back